### ISSUE TITLE
Array title is not rendered

### ISSUE THREAD

**Issue by AlekSi** [Redocly/redoc#319]
```json
"some_param": {
  "type": "array",
  "title": "some_param description",
  "items": {
    "type": "string"
  }
}
```

is rendered as

![2017-08-11_19-56-23](https://user-images.githubusercontent.com/11512/29223425-2d94070a-7ecf-11e7-8891-7cd3b9cb599b.png)

I would like to see "some_param description" somewhere.

Otherwise, ReDoc works like a charm for us. Thank you very much.

**Comment by RomanGotsiy** [comment]
> I would like to see "some_param description" somewhere.

hey @AlekSi, do you have any idea where exactly?

**Comment by AlekSi** [comment]
Not really :) But it should be visible.

**Comment by gerbil** [comment]
What do you mean 'where exactly'? Of course on mouseover! Right now i can see only 'Test' and 'oops' there..

**Comment by phiggins** [comment]
This doesn't appear to be an issue with the latest version (2.0.0-rc14).

**Comment by lornajane** [comment]
This was fixed in the 2.0 release, but let us know if anything still doesn't look right.
